packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -503,25 +513,46 @@ function MapAnalysisImpl(props: ImplProps) {
         })),
       initialDataObject
     );
+    const totalCount = finalNonProportionModeData.reduce(
+      (p, c) => p + c.count,
+      0
+    );
     if (markerType === 'pie') {
       return (
         <DonutMarkerStandalone
-          data={finalData}
-          markerLabel={kFormatter(finalData.reduce((p, c) => p + c.value, 0))}
+          data={finalNonProportionModeData}
+          markerLabel={kFormatter(totalCount)}
           {...sharedStandaloneMarkerProperties}
         />
       );
     } else {
+      const dependentAxisLogScale =
+        activeMarkerConfiguration &&
+        'dependentAxisLogScale' in activeMarkerConfiguration
+          ? activeMarkerConfiguration.dependentAxisLogScale
+          : false;
+      const isPlotModeProportion =
+        activeMarkerConfiguration &&
+        'selectedPlotMode' in activeMarkerConfiguration &&
+        activeMarkerConfiguration.selectedPlotMode === 'proportion';
+      const finalChartMarkerData = isPlotModeProportion
```

Comment:
There's no real need for this ternary switch statement.  The preview marker will look exactly the same if we divide all the counts by `totalCount` or not. The preview marker has no mouse-overs or numbers/axes displayed so it's really just a matter of coloured pixels.

We could keep this in if it feels better though!

Reply:
I went ahead and removed this logic.